pigeonhole.rb

```diff
@@ -30,6 +30,14 @@
   redirect "/noise-candidates/#{today}/#{today}"
 end
 
+def search_query_input 
+  if @search
```

Comment:
Really? Is this opinionated or what makes this cleaner?
The style-guide says to prefer this:
`@search ?  { :conditions => "and incident_key =~ /.*#{@search.strip}.*/i" } : {}`